plugins/UM3NetworkPrinting/NetworkPrinterOutputDevice.py

```diff
@@ -294,6 +294,15 @@ def _setTargetBedTemperature(self, temperature):
         put_request.setHeader(QNetworkRequest.ContentTypeHeader, "application/json")
         self._manager.put(put_request, data.encode())
 
+    ##  Updates the target bed temperature from the printer, and emit a signal if it was changed.
+    #
+    #   /param temperature The new target temperature of the bed.
+    def _updateTargetBedTemperature(self, temperature):
```

Comment:
The first few lines in `_setTargetBedTemperature()` does the same thing here. Could you replace those code in `_setTargetBedTemperature()` with this function call as well?